src/main/java/it/gov/pagopa/payhub/activities/activity/VerifyAuthorizationInstallmentsActivity.java

```diff
@@ -0,0 +1,30 @@
+package it.gov.pagopa.payhub.activities.activity;
```

Comment:
this repo will contains many activities handling many useCases/domain entities, maybe it's better to start to partitionate them
```suggestion
package it.gov.pagopa.payhub.activities.activity.debtposition;
```